ui/pl.json

```diff
@@ -0,0 +1,24 @@
+{
+  "building-delete": "Usuń",
+  "building-move": "Przesuń",
+  "contact_completed": "Kontrakt został wypełniony.",
+  "contact_unlocked": "Nowy kontrakt został odblokowany.",
+  "equipment_unknown": "Błąd: Nieznany ekwipunek.", //Review and context needed here!
+  "equipment_type_unknown": "Błąd: Nieznany typ ekwipunku.", //Review and context needed here!
+  "equipment_space_already_occupied": "Błąd: To miejsce zostało już zajęte.", //Review and context needed here!
+  "equipment_missing_resources": "Błąd: Za mała ilość zasobów aby wyprodukować ten rodzaj ekwipunku.", //Review and context needed here!
+  "market_exchange_no_enough_money": "Błąd: Za mało pieniędzy na tę transakcję.",
+  "market_exchange_no_enough_resource": "Błąd: Za mało zasobów na tę transakcję.", //Review and context needed here!
+  "market_exchange_no_enough_space": "Błąd: Za mało miejsca na tę transakcję.",
+  "popup-close": "Zamknij",
+  "loading-game": "Ładowanie gry...",
+  "money-format": "$%amount%",
```

Comment:
Little note here, the format here is depending on the language, whether you put the "$" before or after the amount in your language. In English that's before, in French after 😄

Reply:
Oh! In that case I'll move it before the dollar sign. Thanks!

Reply:
And same for the buy/sell texts under this :)